packages/amplify-cli-core/API.md

```diff
@@ -491,6 +694,9 @@ export class CloudformationProviderFacade {
     static prePushCfnTemplateModifier(context: $TSContext, template: Template_2): Promise<(template: Template_2) => Promise<void>>;
 }
 
+// @public (undocumented)
+export type CognitoConfiguration = ServiceQuestionsBaseResult & OAuthResult & SocialProviderResult & IdentityPoolResult & PasswordPolicyResult & PasswordRecoveryResult & MfaResult & AdminQueriesResult & Triggers;
+
```

Comment:
We shouldn't move category specific types into core. I.e. category auth _using_ core is fine, but category auth _implementing_ (and also _using_...) core doesn't seem to be right component split.

I think we should:
- either create a new package for category api interfaces (best and it creates nice pattern for other categories too)
- or move these types to amplify-shared-interfaces instead (not ideal, but much better than moving them to core).

Reply:
See https://github.com/aws-amplify/amplify-cli/pull/11962#discussion_r1108848638